devilizer/sorting/bubble_sort.py

```diff
@@ -0,0 +1,19 @@
+def bubble_sort(in_lst: list) -> list:
+    """Bubble Sort Algorithm
+
+    Args:
+        in (list): List to sort.
+
+    Returns:
+        list: Sorted list.
+    """
+    out_lst = in_lst.copy()
+    changes = 1
+    while changes:
```

Comment:
Why not use a boolean here? Using an int seems unintuitive.

Reply:
Hmm, I think it was for ease of use, we could change it too `while changes != 0:`

Reply:
Ah, I don't think I conveyed my point. I meant using a boolean variable instead of an int, e.g.
```py
while not done:
    ...
```